web/packages/teleport/src/UnifiedResources/Resources.tsx

```diff
@@ -311,41 +316,41 @@ export function Resources() {
         setSort={setSort}
         pathname={pathname}
         replaceHistory={replaceHistory}
-        selectAll={toggleSelectAll}
+        selectVisible={toggleSelectVisible}
         selected={allSelected}
         shouldUnpin={shouldUnpin}
       />
       <Flex gap={4} mb={3}>
         {tabs.map(tab => (
           <ResourceTab
             key={tab.value}
+            onClick={() => selectTab(tab.value)}
             title={tab.label}
-            value={tab.value}
             isSelected={
               params.pinnedOnly
                 ? tab.value === UnifiedTabPreference.Pinned
                 : tab.value === UnifiedTabPreference.All
             }
-            onChange={selectTab}
           />
         ))}
       </Flex>
       <ResourcesContainer className="ResourcesContainer" gap={2}>
-        {resources.map(res => {
-          const key = resourceKey(res);
-          return (
-            <ResourceCard
-              key={key}
-              resource={res}
-              onLabelClick={onLabelClick}
-              pinResource={handlePinResource}
-              pinned={pinnedResources.includes(key)}
-              pinningNotSupported={pinningNotSupported}
-              selected={selectedResources.includes(key)}
-              selectResource={handleSelectResources}
-            />
-          );
-        })}
+        {clusterPreferencesAttempt.status !== 'processing' &&
```

Comment:
When normal request is processing we are displaying `LoadingCard`, shouldn't we do the same for this attempt?

Reply:
We should probably make the conditional to show the loading cards based on an OR of normal request processing or cluster preferences processing. both those requests happen 'in parallel' so the loading makes sense, but the cards should only be displayed if the clusterPreferencesAttempt isn't processing (and let the normal request do its thing as normal)

Reply:
Yeah, OR would work here.